js/parts/Series.js

```diff
@@ -4084,22 +4125,35 @@ H.Series = H.seriesType(
             } else {
                 sharedClipKey = this.sharedClipKey;
                 clipRect = chart[sharedClipKey];
-                if (clipRect) {
-                    clipRect.animate({
+
+                if (series.options.clip === false && xAxis) {
+                    // support for not clipped series animation (#10450)
+                    finalBox = inverted ? {
+                        width: chart.chartHeight,
+                        x: -chart.chartHeight + xAxis.pos + xAxis.len
+                    } : {
+                        width: chart.chartWidth,
+                        x: -xAxis.pos
+                    };
+                } else {
+                    finalBox = {
                         width: chart.plotSizeX,
                         x: 0
-                    }, animation);
+                    };
```

Comment:
Can we use the new `getClipBox` for this finalBox as well? In the doclet it says it can be called twice, but it is actually only called once.

Reply:
Yes, done in the commit below. The doclet description is fixed too.